### PR TITLE
E2E QA fixes

### PR DESCRIPTION
**Pull request by ebramirez**

### CODE REVIEW

File: develop-native/solutions/expense-function/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-quarkus.native.builder-image=registry.ocp4.example.com:8443/redhattraining/ubi-quarkus-mandrel-builder-image:jdk-21
+quarkus.native.builder-image=registry.ocp4.example.com:8443/quarkus/ubi-quarkus-mandrel-builder-image:jdk-21
```
Comment (psolarvi) [inline develop-native/solutions/expense-function/src/main/resources/application.properties:1]: Why this change?

Comment (ebramirez) [reply develop-native/solutions/expense-function/src/main/resources/application.properties:1]: Because the script that syncs the images uses the same directory as the upstream one to replicate.

Comment (psolarvi) [reply develop-native/solutions/expense-function/src/main/resources/application.properties:1]: Cool, nice catch!

**Review by psolarvi — APPROVED** [review]
